lib/NormalModule.js

```diff
@@ -467,6 +493,7 @@ class NormalModule extends Module {
 					: getContextify()(context, request);
 			}
 		};
+		/** @type {any} */
```

Comment:
This is necessary because `loaderContext` is modified prior to being returned and TypeScript cannot determine the nature of that modification.